docs/architecture/adr-020-protobuf-transaction-encoding.md

```diff
@@ -28,7 +28,7 @@ querying. However, this ADR solely focuses on transactions. Querying should be
 addressed in a future ADR, but it should build off of these proposals.
 
 Based on detailed discussions ([\#6030](https://github.com/cosmos/cosmos-sdk/issues/6030)
-and [\#6078](https://github.com/cosmos/cosmos-sdk/issues/6078)), the original 
+and [\#6078](https://github.com/cosmos/cosmos-sdk/issues/6078)), the original
```

Comment:
sorry, spacing changes are made by my editor... but i think they make sense.

Reply:
Thank you editor. By the way, have you considered adding an .editorconfig to this repo? I made very good experience with it and use it in all projects: https://editorconfig.org/